### PR TITLE
Script version no longer displayed in Firefox 40+

### PR DESCRIPTION
**Pull request by janekptacijarabaci**
Ad #2249
The suggestion (for example).

But it probably isn't ideal... (name => name+version)

### CODE REVIEW

**Comment by janekptacijarabaci** [comment]
See also https://addons.mozilla.org/cs/firefox/addon/amversionnumber/

If it will be incorporated into the new version, I recommend contacting the author of the extension:

__content/addonversion.css__:

From:
```
.addon:not([type="userstyle"])[status="installed"]
```

To:
```
.addon:not([type="userstyle"]):not([type="greasemonkey-user-script"])[status="installed"]
```

**Comment by ray1138** [comment]
https://addons.mozilla.org/cs/firefox/addon/amversionnumber/

File: content/addons4-overlay.js [hunk 1]
```diff
@@ -24,15 +31,19 @@ var stringBundle = Components
...
+
 // Patch the default createItem() to add our custom property.
 var _createItemOrig = createItem;
 createItem = function GM_createItem(aObj, aIsInstall, aIsRemote) {
   var item = _createItemOrig(aObj, aIsInstall, aIsRemote);
   if (SCRIPT_ADDON_TYPE == aObj.type) {
-   // Save a reference to this richlistitem on the Addon object, so we can
-   // fix the 'executionIndex' attribute if/when it changes.
-   aObj.richlistitem = item;
-   setRichlistitemExecutionIndex(aObj);
```
Comment (ray1138) [inline content/addons4-overlay.js]: https://addons.mozilla.org/cs/firefox/addon/amversionnumber/

**Comment by arantius** [comment]
I think I'd prefer to let users install (as linked) an extension to completely restore this feature, if they want it.  Consistently not display the version number, if they don't.

**Comment by janekptacijarabaci** [comment]
If it's not used, you can close it.

**Comment by janekptacijarabaci** [comment]
See also https://github.com/greasemonkey/greasemonkey/issues/2249#issuecomment-139538843

**Pull request closed without merging**